data/typelibrary/devices-1.0.0/typelib/UAO_RT.dev

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE DeviceType SYSTEM "http://www.holobloc.com/xml/LibraryElement.dtd" >
+<DeviceType Name="UAO_RT" Comment="UAO Runtime running on a PC (Windows or Linux)" >
+  <Identification Standard="61499 ITA" Description="Copyright (c) 2017 Aimirim STI" />
```

Comment:
please add the reference to the eclipse license into the description field like the other devices and function blocks.

Reply:
License notice added in the description after the 'Copyright' in a40b827